reviews/models/mixins.py

```diff
@@ -244,30 +245,15 @@ def get_context(self):
 
 # Handle email notifications including: update comment, accept, and reject of submission.
 @reviews_signals.reviews_email.connect
-def reviews_notification(self, context):
-    timestamp = timezone.now()
-    event_type = utils.find_subscription_type('global_reviews')
-    template = context['template'] + '.html.mako'
-    for user_id in context['email_recipients']:
-        user = OSFUser.load(user_id)
-        subscriptions = get_user_subscriptions(user, event_type)
-        for notification_type in subscriptions:
-            check_user_subscribe = subscriptions[notification_type] and user_id in subscriptions[notification_type] and notification_type != 'none'  # check if user is subscribed to this type of notifications
-            if check_user_subscribe:
-                node_lineage_ids = get_node_lineage(context.get('reviewable').node) if context.get('reviewable').node else []
-                context['user'] = user
-                context['no_future_emails'] = notification_type == 'none'
-                send_type = notification_type if notification_type != 'none' else 'email_transactional'
-                message = mails.render_message(template, **context)
-                digest = NotificationDigest(
-                    user=user,
-                    timestamp=timestamp,
-                    send_type=send_type,
-                    event='global_reviews',
-                    message=message,
-                    node_lineage=node_lineage_ids
-                )
-                digest.save()
+def reviews_notification(self, context, node):
+    user = context['email_sender']
+    auth = Auth(user)
+    time_now = timezone.now()
+    email_recipients = context['email_recipients']
+    email_recipients.remove(user._id)  # remove email sender
+    for user_id in email_recipients:
```

Comment:
`notify` finds users who are subscribed to an event on a node. It seems like it's meant to be called just once for the event, not once per email recipient.